cybox/bindings/win_task_object.py

```diff
@@ -1523,7 +1522,7 @@ def buildChildren(self, child_, node, nodeName_, fromsubclass_=False):
     'Fuzzy_Hash_Value': cybox_common.FuzzyHashValueType,
     'Show_Message_Title': cybox_common.StringObjectPropertyType,
     'Data_Size': cybox_common.DataSizeType,
-    'Email_Message': email_message_object.EmailMessageObjectType,
+    'email_message': email_message_object.EmailMessageObjectType,
```

Comment:
I think it is a find-replace gone bad. It pops up in a couple spots. I fixed it in 8327de6e1f60b32050910abcba88d24e6bd87e5f.